Quadriga/src/main/java/edu/asu/spring/quadriga/rest/open/NodeInfoController.java

```diff
@@ -26,26 +26,27 @@
 @PropertySource(value = "classpath:/settings.properties")
 @Controller
 public class NodeInfoController {
-    
+
     @Autowired
     private INetworkManager networkManager;
-    
+
     @Autowired
     private Environment env;
 
-
     @RequestMapping(value = "public/concept/texts", method = RequestMethod.GET)
-    public ResponseEntity<String> getTextsForConcepts(@RequestParam String conceptId, @RequestParam String projectUnix) throws Exception {
+    public ResponseEntity<String> getTextsForConcepts(@RequestParam String conceptId, @RequestParam String projectUnix)
+            throws Exception {
         Set<TextOccurance> occurances = networkManager.getTextsForConceptId(conceptId, ETextAccessibility.PUBLIC);
         JSONArray projectTexts = new JSONArray();
         JSONArray otherProjectsTexts = new JSONArray();
-        
+
         for (TextOccurance occur : occurances) {
-            if (occur.getProject().getUnixName().equals(projectUnix.trim())) {
+            if (occur.getProject().getUnixName().equals(projectUnix.trim())
+                    || occur.getProject().getUnixName().equals("unkown")) {
```

Comment:
where is this coming from? why is this checking for a project with unix name "unkown" (which is misspelled btw).